internal/commands/parameters.go

```diff
@@ -45,6 +47,22 @@ func withCommandLineParameters(overrides []string) mergeBundleOpt {
 	}
 }
 
+func withLabels(labels []string) mergeBundleOpt {
```

Comment:
Don't we want to reserve labels with `com.docker.app.` prefix for the docker app internal use only?

Reply:
Maybe we should not allow labels whose key is prefixed with `com.docker.app.` to be set with the `--label` flag and keep these labels for internal (and also future) docker App use.